examples/aws/06-ark-volumesnapshotlocation.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 the Heptio Ark contributors.
```

Comment:
Please add similar example files to `examples/azure` and `examples/gcp`, with the appropriate config variables